doc/python/axes.md

```diff
@@ -380,6 +380,26 @@ fig.update_layout(title_text="Apple Stock Price")
 fig.show()
 ```
 
+#### Adding minor ticks
+You can add minor ticks to an axis by specifying with the `minor` property. This takes a `dict` of properties to apply to minor ticks. 
```

Comment:
please add a line here (and in every new example for this release) in italics with "New in v5.8" or similar :)

Reply:
Done